components/layout/search/filter/dropdown.tsx

```diff
@@ -45,7 +45,7 @@ export default function FilterItemDropdown({ list }: { list: ListItem[] }) {
         className="flex w-full items-center justify-between rounded border border-black/30 px-4 py-2 text-sm dark:border-white/30"
       >
         <div>{active}</div>
-        <Caret className="h-4 rotate-90" />
+        <ChevronRightIcon className="h-4" />
```

Comment:
This is pointing the incorrect direction. 

![CleanShot 2023-07-10 at 15 13 02@2x](https://github.com/vercel/commerce/assets/446260/a60b75e8-08b3-40e2-adbf-71f0dc1c4e51)